rasgoql/rasgoql/data/postgres.py

```diff
@@ -0,0 +1,609 @@
+"""
+Snowflake DataWarehouse classes
+"""
+import logging
+import os
+from typing import List, Optional, Union
+
+import json
+import pandas as pd
+
+from rasgoql.data.base import DataWarehouse, DWCredentials
+from rasgoql.errors import (
+    DWCredentialsWarning, DWConnectionError, DWQueryError,
+    PackageDependencyWarning, ParameterException,
+    SQLWarning, TableAccessError, TableConflictException
+)
+from rasgoql.imports import alchemy_engine, alchemy_text, alchemy_session
+from rasgoql.primitives.enums import (
+    check_response_type, check_table_type, check_write_method
+)
+from rasgoql.utils.creds import load_env, save_env
+from rasgoql.utils.df import cleanse_sql_dataframe, generate_dataframe_ddl
+from rasgoql.utils.messaging import verbose_message
+from rasgoql.utils.sql import (
+    is_scary_sql, magic_fqtn_handler,
+    parse_fqtn, parse_namespace,
+    validate_namespace, validate_fqtn, parse_table_and_schema_from_fqtn
+)
+
+logging.basicConfig()
+logger = logging.getLogger('Postgres DataWarehouse')
+logger.setLevel(logging.INFO)
+
+
+class PostgresCredentials(DWCredentials):
+    """
+    Postgres Credentials
+    """
+    dw_type = 'postgresql'
+
+    def __init__(
+            self,
+            username: str,
+            password: str,
+            host: str,
+            port: str,
+            default_db: str,
```

Comment:
Minor style thing: Snowflake and BQ creds omit the `default_` and just use `database` or `project`. For consistency: should we do the same here? or make those params `default_`?